[2025-04-15T09:40:38.974+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-15T09:40:39.013+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-15T07:20:00+00:00 [queued]>
[2025-04-15T09:40:39.028+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-15T07:20:00+00:00 [queued]>
[2025-04-15T09:40:39.028+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-04-15T09:40:39.060+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): run_dbt_trusted> on 2025-04-15 07:20:00+00:00
[2025-04-15T09:40:39.066+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3991) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-15T09:40:39.067+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'extract_raw_data_pipeline', 'run_dbt_trusted', 'scheduled__2025-04-15T07:20:00+00:00', '--job-id', '113', '--raw', '--subdir', 'DAGS_FOLDER/lead_data_pipeline.py', '--cfg-path', '/tmp/tmp94q_fw5d']
[2025-04-15T09:40:39.069+0000] {standard_task_runner.py:72} INFO - Started process 3993 to run task
[2025-04-15T09:40:39.069+0000] {standard_task_runner.py:105} INFO - Job 113: Subtask run_dbt_trusted
[2025-04-15T09:40:39.129+0000] {task_command.py:467} INFO - Running <TaskInstance: extract_raw_data_pipeline.run_dbt_trusted scheduled__2025-04-15T07:20:00+00:00 [running]> on host c0ac39583bd8
[2025-04-15T09:40:39.253+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='extract_raw_data_pipeline' AIRFLOW_CTX_TASK_ID='run_dbt_trusted' AIRFLOW_CTX_EXECUTION_DATE='2025-04-15T07:20:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-15T07:20:00+00:00'
[2025-04-15T09:40:39.254+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-15T09:40:39.280+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-04-15T09:40:39.284+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -x; cd /opt/***/dbt && dbt run --select tag:trusted']
[2025-04-15T09:40:39.294+0000] {subprocess.py:86} INFO - Output:
[2025-04-15T09:40:39.296+0000] {subprocess.py:93} INFO - + cd /opt/***/dbt
[2025-04-15T09:40:39.297+0000] {subprocess.py:93} INFO - + dbt run --select tag:trusted
[2025-04-15T09:40:43.618+0000] {subprocess.py:93} INFO - [0m09:40:43  Running with dbt=1.8.0
[2025-04-15T09:40:46.014+0000] {subprocess.py:93} INFO - [0m09:40:46  Registered adapter: postgres=1.8.2
[2025-04-15T09:40:49.735+0000] {subprocess.py:93} INFO - [0m09:40:49  Found 9 models, 10 sources, 428 macros
[2025-04-15T09:40:49.769+0000] {subprocess.py:93} INFO - [0m09:40:49
[2025-04-15T09:40:51.011+0000] {subprocess.py:93} INFO - [0m09:40:51  Concurrency: 1 threads (target='dev')
[2025-04-15T09:40:51.020+0000] {subprocess.py:93} INFO - [0m09:40:51
[2025-04-15T09:40:51.161+0000] {subprocess.py:93} INFO - [0m09:40:51  1 of 4 START sql table model lead_trusted.non_pii_data ......................... [RUN]
[2025-04-15T09:56:23.531+0000] {job.py:229} INFO - Heartbeat recovered after 928.93 seconds
[2025-04-15T10:09:28.938+0000] {job.py:229} INFO - Heartbeat recovered after 744.39 seconds
[2025-04-15T10:24:55.524+0000] {job.py:229} INFO - Heartbeat recovered after 905.88 seconds
[2025-04-15T10:40:46.607+0000] {job.py:229} INFO - Heartbeat recovered after 930.47 seconds
[2025-04-15T10:56:36.063+0000] {job.py:229} INFO - Heartbeat recovered after 902.94 seconds
[2025-04-15T11:10:29.929+0000] {job.py:229} INFO - Heartbeat recovered after 813.37 seconds
[2025-04-15T11:10:38.964+0000] {subprocess.py:93} INFO - [0m11:10:38  1 of 4 OK created sql table model lead_trusted.non_pii_data .................... [[32mSELECT 21225910[0m in 5387.80s]
[2025-04-15T11:10:38.969+0000] {subprocess.py:93} INFO - [0m11:10:38  2 of 4 START sql table model lead_trusted.payment_data ......................... [RUN]
[2025-04-15T11:26:21.104+0000] {job.py:229} INFO - Heartbeat recovered after 930.65 seconds
[2025-04-15T11:42:32.778+0000] {job.py:229} INFO - Heartbeat recovered after 930.48 seconds
[2025-04-15T11:42:33.493+0000] {subprocess.py:93} INFO - [0m11:42:33  2 of 4 OK created sql table model lead_trusted.payment_data .................... [[32mSELECT 3977366[0m in 1914.52s]
[2025-04-15T11:42:33.497+0000] {subprocess.py:93} INFO - [0m11:42:33  3 of 4 START sql table model lead_trusted.pii_data ............................. [RUN]
[2025-04-15T11:58:46.952+0000] {job.py:229} INFO - Heartbeat recovered after 927.82 seconds
[2025-04-15T12:11:30.850+0000] {job.py:229} INFO - Heartbeat recovered after 722.77 seconds
[2025-04-15T12:27:43.245+0000] {job.py:229} INFO - Heartbeat recovered after 931.27 seconds
[2025-04-15T12:43:55.690+0000] {job.py:229} INFO - Heartbeat recovered after 931.15 seconds
[2025-04-15T12:59:47.782+0000] {job.py:229} INFO - Heartbeat recovered after 904.87 seconds
[2025-04-15T13:12:33.095+0000] {job.py:229} INFO - Heartbeat recovered after 743.86 seconds
[2025-04-15T13:28:23.745+0000] {job.py:229} INFO - Heartbeat recovered after 930.08 seconds
[2025-04-15T14:13:34.027+0000] {job.py:229} INFO - Heartbeat recovered after 2689.59 seconds
[2025-04-15T14:29:29.080+0000] {job.py:229} INFO - Heartbeat recovered after 929.33 seconds
[2025-04-15T14:45:19.562+0000] {job.py:229} INFO - Heartbeat recovered after 904.34 seconds
[2025-04-15T15:00:46.713+0000] {job.py:229} INFO - Heartbeat recovered after 906.60 seconds
[2025-04-15T15:14:35.867+0000] {job.py:229} INFO - Heartbeat recovered after 808.57 seconds
[2025-04-15T15:14:48.512+0000] {subprocess.py:93} INFO - [0m15:14:48  3 of 4 OK created sql table model lead_trusted.pii_data ........................ [[32mSELECT 21225910[0m in 12735.01s]
[2025-04-15T15:14:48.528+0000] {subprocess.py:93} INFO - [0m15:14:48  4 of 4 START sql table model lead_trusted.technical_data ....................... [RUN]
[2025-04-15T15:30:24.477+0000] {job.py:229} INFO - Heartbeat recovered after 907.44 seconds
[2025-04-15T15:46:14.426+0000] {job.py:229} INFO - Heartbeat recovered after 929.40 seconds
[2025-04-15T15:46:40.996+0000] {subprocess.py:93} INFO - [0m15:46:40  4 of 4 OK created sql table model lead_trusted.technical_data .................. [[32mSELECT 3154094[0m in 1912.45s]
[2025-04-15T15:46:41.107+0000] {subprocess.py:93} INFO - [0m15:46:41
[2025-04-15T15:46:41.114+0000] {subprocess.py:93} INFO - [0m15:46:41  Finished running 4 table models in 6 hours 5 minutes and 51.33 seconds (21951.33s).
[2025-04-15T15:46:41.527+0000] {subprocess.py:93} INFO - [0m15:46:41
[2025-04-15T15:46:41.536+0000] {subprocess.py:93} INFO - [0m15:46:41  [32mCompleted successfully[0m
[2025-04-15T15:46:41.544+0000] {subprocess.py:93} INFO - [0m15:46:41
[2025-04-15T15:46:41.553+0000] {subprocess.py:93} INFO - [0m15:46:41  Done. PASS=4 WARN=0 ERROR=0 SKIP=0 TOTAL=4
[2025-04-15T15:46:43.395+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-04-15T15:46:43.574+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-15T15:46:43.579+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=extract_raw_data_pipeline, task_id=run_dbt_trusted, run_id=scheduled__2025-04-15T07:20:00+00:00, execution_date=20250415T072000, start_date=20250415T094039, end_date=20250415T154643
[2025-04-15T15:46:43.693+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-15T15:46:43.776+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-15T15:46:43.797+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
